package.json

```diff
@@ -19,6 +19,7 @@
     "@babel/preset-react": "^7.0.0",
     "babel-eslint": "^10.0.3",
     "babel-loader": "^8.0.6",
+    "bootstrap-sass": "^3.4.1",
```

Comment:
I really wonder how it worked at the podman repo!??

Reply:
Oh I know, we don't include there the form-layout.scss that imports the variables.scss file.